src/cpl/share_esmf/ExcessIceStreamType.F90

```diff
@@ -0,0 +1,295 @@
+module ExcessIceStreamType
+
+#include "shr_assert.h"
+
+  !-----------------------------------------------------------------------
+  ! !DESCRIPTION:
+  ! Contains methods for reading in excess ice initial bulk values from data stream
+  !
+  ! !USES
+  use ESMF
+  use dshr_strdata_mod , only : shr_strdata_type
+  use shr_kind_mod     , only : r8 => shr_kind_r8, CL => shr_kind_cl
+  use shr_log_mod      , only : errMsg => shr_log_errMsg
+  use spmdMod          , only : mpicom, masterproc
+  use clm_varctl       , only : iulog
+  use abortutils       , only : endrun
+  use decompMod        , only : bounds_type
+
+  ! !PUBLIC TYPES:
+  implicit none
+  private
+
+  type, public :: excessicestream_type
+     real(r8), pointer, private :: exice_bulk  (:)         ! excess ice bulk value (-)
+  contains
+
+      ! !PUBLIC MEMBER FUNCTIONS:
+      procedure, public :: Init            ! Initialize and read data in
+      procedure, public :: UseStreams      ! If streams will be used
```

Comment:
It looks like UseStreams is actually private, so mark it as such.

Reply:
Fixed